packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -34,22 +34,22 @@ function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLEl
     'aria-labelledby': ariaLabelledby
   } = props;
 
-  let inputRef = useRef<HTMLInputElement | HTMLTextAreaElement>();
-  let domRef = useFocusableRef(ref, inputRef);
+  let domRef = useFocusableRef(ref);
+  let groupRef = useRef<HTMLDivElement>();
 
   // Measure the width of the field to inform the width of the menu.
   let [menuWidth, setMenuWidth] = useState(null);
   let {scale} = useProvider();
 
   let onResize = useCallback(() => {
-    if (domRef.current) {
-      let width = domRef.current.offsetWidth;
+    let width = groupRef.current?.offsetWidth;
```

Comment:
Hm, `no label` still looks like this, `groupRef.current` is coming up as undefined
![image](https://user-images.githubusercontent.com/9661127/127686307-5057d63d-c4e7-4648-b4fd-42665d921a31.png)